src/ui/marker.js

```diff
@@ -31,6 +34,9 @@ type Options = {
  * @param {PointLike} [options.offset] The offset in pixels as a {@link PointLike} object to apply relative to the element's center. Negatives indicate left and up.
  * @param {string} [options.color='#3FB1CE'] The color to use for the default marker if options.element is not provided. The default is light blue.
  * @param {boolean} [options.draggable=false] A boolean indicating whether or not a marker is able to be dragged to a new position on the map.
+ * @param {number} [options.rotation=0] The rotation angle of the marker (clockwise, in degrees), relative to its respective {@link Marker#rotationAlignment} setting.
```

Comment:
As far as I can tell, nothing in this PR restricts rotation to the clockwise direction (which I think is good!). If you set `rotation = -45`, then the marker will rotate 45 degrees counterclockwise, right? So we should spell that out here.

Reply:
Maybe just enough to replace "clockwise, in degrees" with "a positive rotation angle is clockwise, units in degrees"?

Reply:
Maybe `The rotation angle of the marker in degrees, relative to its respective {@link Marker#rotationAlignment} setting. A positive value will rotate the marker clockwise.`

Reply:
Updated the parameter description.